.azure/pipelines/ci.yml

```diff
@@ -565,7 +565,7 @@ stages:
       jobDisplayName: "Build: Linux Musl ARM"
       agentOs: Linux
       useHostedUbuntu: false
-      container: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-arm-alpine-20200827125937-14441ae
+      container: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-cross-arm-alpine-20210409142327-044d5b9
```

Comment:
We do not want to be using anything built on Ubuntu 16.04 after the end of the month; that release is EOL. Is there an equivalent 18.04 docker image❔

Reply:
Besides that, this Docker image doesn't seem to contain the `crossgen` dependencies:
``` text
/__w/1/s/artifacts/obj/Microsoft.AspNetCore.App.Runtime/Release/net6.0/linux-musl-arm/crossgen/crossgen: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.27' not found (required by /__w/1/s/artifacts/obj/Microsoft.AspNetCore.App.Runtime/Release/net6.0/linux-musl-arm/crossgen/crossgen)
```
That worked fine w/ the current image.

Reply:
Hmm, the crossgen dependencies are likely due to the fact that the changes in runtime where I've switched to the 16.04 image too didn't result in new packages that aspnet would use. 
The reason for switching to 16.04 was to make crossgen running on x64 to work on older Linux distros. Ideally, we would crossbuild on CentOS 7, the same OS we use for regular builds. However our crossbuilding doesn't work there due to some differences in include / library paths compiled into the clang compiler, so we have to stick with the oldest Ubuntu possible. I am aware though that the 16.04 is going out of support, so I was going to try to make cross build work on the centos 7 before that happens.
However, for ASPNET core, it doesn't matter where we crossbuild (there are no x64 build artifacts in arm build), so let me use the 18.04 here. 
While I am on it, I can also update the arm64 image to move to arm64 so that we don't need to do another PR at the end of the month if you want.

Reply:
I've actually went ahead and moved both arm and arm64 to the 18.04.